### PR TITLE
#4251 [Manager] Nodes page need display node state

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4251 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/common/Table/SelectTable.js [hunk 1]
```diff
@@ -104,7 +105,7 @@ const SelectTable = props => {
   };
 
   const handleClick = row => {
-    if (!hasSelect || isItemDisabled(row)) return;
+    if (!hasSelect || isItemDisabled(row) || isItemIndeterminate(row)) return;
```
Comment (konekoya) [inline ohara-manager/client/src/components/common/Table/SelectTable.js]: `isItemIndeterminate` what does this mean?

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
@wu87988622 I've run the PR a couple of times and it looks like this PR is breaking one of the node e2e tests?

**Comment by wu87988622** [comment]
> @wu87988622 I've run the e2e test a couple of times and it looks like this PR is breaking one of the node e2e tests?

Thanks , i will check e2e test .

**Comment by wu87988622** [comment]
retry manager-e2e

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**